requirements.txt

```diff
@@ -0,0 +1,7 @@
+setuptools>=45.2.0
+Cython>=0.29.24
+wheel>=0.34.2
+cibuildwheel>=2.8.0
+Sphinx>=5.0.2
+sphinx-rtd-theme>=1.0.0
+tox>=3.25.1
```

Comment:
I'm not sure this is correct:

If I understand correctly requirements is for runtime, whilst we only really care about having these tools (incidentally, tox is no longer required) at build time.

Probably worth explanding:

```
[project.optional-dependencies]
publish = ["twine", "wheel"]
ci = ["cibuildwheel"]
```

and

```
[build-system]
requires = [
    # Setuptools 18.0 and above properly handles Cython extensions.
    "setuptools>=45.2.0",
    "wheel>=0.34.2",
    "cython>=0.29.24"]
```

In `pyproject.toml`.